lib/swagger_client/models/dmca_search_request_v2.rb

```diff
@@ -0,0 +1,272 @@
+=begin
+#TeePublic V3 search Api Documentation
+
+#No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+
+OpenAPI spec version: 1.0.9
```

Comment:
Is this generated by swagger-codegen? Why is the version 1.0.9 instead of 1.0.11? Also I think we need to manually update the version in lib/swagger_client/version.rb

Reply:
@jimmyzhang-tp The updates weren't generated by swagger-codegen. Good catch, thanks.